lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/Transactions.hs

```diff
@@ -210,6 +217,38 @@ spec = do
         expectEventually' ctx getWalletEp balanceAvailable amt w
         expectEventually' ctx getWalletEp balanceTotal amt w
 
+    describe "TRANS_DELETE_05 - Cannot forget external tx -> 404" $ do
```

Comment:
now when http-bridge is nuked we can have this test in core-integration, right?

Reply:
of course, this refers to all tests located here, like external Tx, etc., we can reshuffle in next PR

Reply:
btw. good point. because of how normal tx and external tx is treated, external cannot be forgot in the sense as normal tx

Reply:
> now when http-bridge is nuked we can have this test in core-integration, right?

Yes, some of them sure, but others probably it would be good to leave here. (Like this one, is using jcli to prepare external tx, which is pretty much Jormungandr specific)